### PR TITLE
Support multiple CSS files

### PR DESCRIPTION
**Pull request by darrenburns**
- Allows you to use multiple CSS files in `css_path` and `CSS_PATH`
- Documents interaction between classvar CSS and the loading of external CSS files.
- Allows you to supply paths containing Windows drive names to `import_app` (subsequently also snapshot_tests).
- Some improvements to snapshot testing messaging
  - When you create a snapshot for the first time and it fails, it'll now generate a failure report which shows the screenshot which was taken and a message saying that there was no historical screenshot. It also contains instructions on how to save the new snapshot to disk.

image

### CODE REVIEW

File: docs/guide/CSS.md [hunk 1]
```diff
@@ -158,6 +159,10 @@ With the CSS in place, the output looks very different:
 
 ```
 
+### Using multiple CSS files
+
+You can also set the `CSS_PATH` class variable to a list of paths, and they'll be loaded in order.
```
Comment (willmcgugan) [inline docs/guide/CSS.md:164]:
```suggestion
You can also set the `CSS_PATH` class variable to a list of paths. Textual will combine the rules from all of the supplied paths.
```

File: tests/snapshot_tests/conftest.py [hunk 2]
```diff
@@ -114,14 +124,14 @@ def pytest_sessionfinish(
...
-        if snapshot_svg and actual_svg and app:
+        if app:
             path, line_index, name = item.reportinfo()
             diffs.append(
                 SvgSnapshotDiff(
                     snapshot=str(snapshot_svg),
                     actual=str(actual_svg),
                     file_similarity=100
-                    * difflib.SequenceMatcher(
+                                    * difflib.SequenceMatcher(
```
Comment (willmcgugan) [inline tests/snapshot_tests/conftest.py:134]: Weird formatting? Maybe put the expression in parenthesis. That sometimes prompts black to reformat.

**Review by willmcgugan — APPROVED** [review]
Awesome. Couple of suggestions to consider.

File: tests/snapshot_tests/conftest.py (hunk 2 above)
Comment (darrenburns) [reply tests/snapshot_tests/conftest.py]: Fixed

**Pull request merged**